libs/design/badge/src/badge/badge.component.html

```diff
@@ -0,0 +1,3 @@
+<ng-content select="[daffBadgeIcon]"></ng-content>
```

Comment:
`<ng-content select="[daffBadgeIcon]"></ng-content>` should be replaced with `DaffPrefixabledDirective`.

See the button or notification component for an example on how we do this.